test-framework/junit5/src/main/java/io/quarkus/test/junit/launcher/ConfigUtil.java

```diff
@@ -21,14 +21,18 @@ public static List<String> argLineValue(Config config) {
         if (strValue == null) {
             return Collections.emptyList();
         }
-        String[] parts = strValue.split("\\s+");
+        return splitArgs(strValue);
```

Comment:
In `quarkus-core-deployment` (which should be available), we have `CommandLineUtil#translateCommandline()` and I think it might be a better fit for this.
WDYT?

Reply:
@metacosm I'm back on this and I'm wondering why the difference would be a problem? If you pass the arguments as separate entities, I think they should be parsed correctly even without the quotes?

Or do you see a problem with that?